### PR TITLE
UVify Core - verified by UVify

### PR DESCRIPTION
**Pull request by hyonlim**
**Describe problem solved by this pull request**
We have verified Draco-R, IFO and Draco for this PR. We will consider to provide vehicles to test team going forward.

**Describe your solution**
3 Platforms are fully tested.

### CODE REVIEW

**Comment by dagar** [comment]
I think the bootloader is ending up in the ROMFS for other boards. We need to make sure it's stored per board.

**Comment by dagar** [comment]
How about we add the bootloader binary to a `bootloader/` folder within the board and specify it in the board without the path?

https://github.com/PX4/Firmware/blob/e843090383e827f00830767a3836b1fa7fcc5d27/boards/px4/fmu-v2/default.cmake#L10


Then ROMFS can copy it during the build.
https://github.com/PX4/Firmware/blob/e843090383e827f00830767a3836b1fa7fcc5d27/ROMFS/CMakeLists.txt#L141-L147

File: ROMFS/px4fmu_common/extras/README.txt [hunk 1]
```diff
@@ -3,3 +3,7 @@ px4fmuv3_bl.bin: Bootloader binary of the PX4 Bootloader
   based on commit 184b813699a9cfd6f43a5a21556a06b4372baf5f
   for the target px4fmuv3_bl
 
+uvify_core_bl.bin: Bootloader binary of the UVify Core Bootloader
+  (https://github.com/uvify/Bootloader)
+  based on commit 60ba275537aa3b777ea68d419c2d0dbe25713c7f
```
Comment (bkueng) [inline ROMFS/px4fmu_common/extras/README.txt]: Can you open a pull request for the bootloader as well?

**Comment by hyonlim** [comment]
Suggestions are reflected. This is another PR.
https://github.com/PX4/Firmware/pull/13317

**Pull request closed without merging**